arboretum/ibm_cloud/README.md

```diff
@@ -18,10 +18,62 @@ how to include the fetchers and checks from this library in your downstream proj
 
 ## Fetchers
 
-Fetchers coming soon...
+### Cluster List
+
+* Class: [ClusterListFetcher][fetch-cluster-list]
+* Purpose: Write the list of IBM Cloud clusters to the evidence locker.
+* Behavior: Log in to IBM Cloud using `ibmcloud login` command, and save the result of `ibmcloud cs cluster ls` command.
+* Expected configuration elements:
+  * org.ibm_cloud.cluster_list.config
+    * List of objects representing the IKS accounts
+    * Each object must have the following values
+      * `account` - a list containing names identifying the IKS account, this will map to an IAM token provided in the credentials file
+* Expected configuration example:
+   ```json
+   {
+     "org": {
+       "ibm_cloud": {
+         "cluster_list": {
+           "config": {
+               "account": ["myaccount1"]
+           }
+         }
+       }
+     }
+   }
+   ```
+* <a name="expected_credentials"></a>Expected credentials:
```

Comment:
Same as above, I think we're using "Required credentials:" now and I don't think this needs to be an anchor (yet).

Reply:
removed anchor.